apps/editor.planx.uk/src/pages/FlowEditor/components/NewSettings/shared/SettingsForm.tsx

```diff
@@ -92,16 +94,16 @@ const SettingsFormContainer = <
   });
 
   const formik = useFormik<TFormValues>({
-    initialValues: data
-      ? getInitialValues(data)
-      : (validationSchema.getDefault() as TFormValues),
```

Comment:
Even though this worked to populate values, it turns out yup / formik were also stubbing these values in when running validation. This meant it was impossible to hit a validation error.

Using a a new `defaultValues` prop is actually more type safe - there's no way of enforcing a yup schema has `.default("something")` appended to each field.